src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    const arr = this;
+    let result = '';
+
+    for (let i = 0; i < arr.length; i++) {
+      if (arr[i] !== null && arr[i] !== undefined) {
```

Comment:
As I know eslint won't allow you to do it, but just FYI:

There is operator such `??` which would be great to use here